getting-started/organization-limits.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Organization Limits"
```

Comment:
Is it worth including this section at all, or should we just keep the "Maximum Users Per Org" section and wait on how-tos?